extensions/wikia/AdEngine/js/ml/model/logisticRegression.js

```diff
@@ -0,0 +1,30 @@
+/*global define*/
+define('ext.wikia.adEngine.ml.model.logisticRegression', [], function () {
+	'use strict';
+
+	function create(coefficients, intercept) {
+		function predict(x) {
+			var i,
+				result = 0;
+
+			if (x.length !== coefficients.length) {
+				throw 'Incorrect vectors';
```

Comment:
I think we should catch it as some level. Otherwise we stop JS execution for the rest of file.